Fix WebSocket Warnings

Please fix these warnings, and enable "Warnings as errors" in the relevant project. 

> 55>WebSocketTransportTests.cs(30,13,30,32): warning CS4014: Because this call is not awaited, execution of the current method continues before the call is completed. Consider applying the 'await' operator to the result of the call.
> 55>WebSocketTransportTests.cs(25,27,25,51): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>WebSocketTransportTests.cs(35,27,35,59): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>WebSocketTransportTests.cs(52,27,52,56): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>WebSocketTransportTests.cs(94,27,94,54): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>WebSocketTransportTests.cs(107,27,107,55): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>WebSocketTransportTests.cs(192,64,192,76): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.
> 55>Mocks\MockPayloadSender.cs(23,47,23,59): warning CS0067: The event 'MockPayloadSender.Disconnected' is never used
>

--- a/tests/Microsoft.Bot.Streaming.Tests/Mocks/MockPayloadSender.cs
+++ b/tests/Microsoft.Bot.Streaming.Tests/Mocks/MockPayloadSender.cs
@@ -20,7 +20,9 @@ namespace Microsoft.Bot.Streaming.UnitTests
             SentHeaders = new List<Header>();
         }
 
+#pragma warning disable CS0067
         public event DisconnectedEventHandler Disconnected;
+#pragma warning restore CS0067
 
         public List<Header> SentHeaders { get; set; }
 

--- a/tests/Microsoft.Bot.Streaming.Tests/WebSocketTransportTests.cs
+++ b/tests/Microsoft.Bot.Streaming.Tests/WebSocketTransportTests.cs
@@ -4,16 +4,12 @@
 
 using System;
 using System.Collections.Generic;
-using System.IO.Pipes;
-using System.Net.Mail;
-using System.Net.Sockets;
 using System.Net.WebSockets;
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.Bot.Builder.Integration.AspNet.Core;
 using Microsoft.Bot.Builder.Streaming;
-using Microsoft.Bot.Streaming.Transport.NamedPipes;
 using Microsoft.Bot.Streaming.Transport.WebSockets;
 using Microsoft.Bot.Streaming.UnitTests.Mocks;
 using Xunit;

